Cross organization deployment (#309)
awslabs/aws-deployment-framework: docs/user-guide.md

    properties: ...
```

> It is also possible to target accounts outside the current Organization by giving *ext-account-id://* key on the targets definition as shown below. Before doing this make sure that the target account is bootstrapped and the target organization has the permissions to use the KMS key and the S3 buckets. This needs to be done manually. More information about how to bootstrap manually can be found at the *admin-guide*.

Reviewer: ```suggestion
It is also possible to target accounts outside the current Organization by supplying the account id prefixed with the `ext-account-id://` key on the targets definition as shown below. Before doing this, make sure that the target account is bootstrapped and the target organization has the permissions to use the KMS key and the S3 buckets. This needs to be done manually. More information about how to bootstrap manually can be found in the [admin guide](./admin-guide.md#bootstrapping-accounts-outside-the-current-organization).
```
